### DISCUSSION TITLE
How to make enter key act as a tab instead of submit form?

### DISCUSSION THREAD

**Discussion by theoreaves** [livewire/livewire#3838] [1. Help] [answered]
I know this can be done in just straight javascript, but is there a way to do this with Alpine/Livewire?  I basically want to catch the enter key, stop the submit and then tab to the next field.

Thanks!

**Comment by ryatkins** [comment, marked as answer]
What does your HTML look like right now?

As this is not standard browser behavior you would need to remove your input out of a form tag so that the front end isn't trying to submit by default on an enter key.

Then you would handle it with Alpine.  Here is some pseudo code, but to do this right you'll need to gather all your inputs and get the next one via javascript.

```
<input @keydown.enter="nextInput" />

<input x-ref="myNextInput" />

```

```
nextInput () {
    this.$nextTick(() => (
                    this.$refs.myNextInput.focus()
                ));
}
```

**Reply by theoreaves** [reply]
I did have it wrapped in form tags.  I've removed the form tags and now have a working solution.  Thanks!
